enterprise/frontend/src/metabase-enterprise/sdk-plugins.ts

```diff
@@ -29,6 +30,7 @@ export function initializePlugins() {
   initializeWhitelabelOverridePlugin?.();
   initializeContentTranslation?.();
   initializeSubscriptions();
+  initializeSharing();
```

Comment:
Here's the fix. It's unsed in both the modular embedding and modular embedding SDK.